README.md

```diff
@@ -1,5 +1,12 @@
 # wikibase-seed-data
 
+## Setup
+
+Run `npm install` to install dependencies, e.g.:
+```bash
+docker-composer run node npm install
```

Comment:
Hah, this is one of my favorite things to typo too.

```suggestion
docker-compose run node npm install
```

Reply:
Ahh yes; that old chestnut! Won't be the last time I make that mistake xD